src/main/java/com/ghilly/service/CountryService.java

```diff
@@ -1,9 +1,16 @@
 package com.ghilly.service;
 
-public class CountryService {
 
+import java.util.List;
 
-    public String serviceSaysHello() {
-        return "Hello, I'm Service";
-    }
+public interface CountryService {
+        String add(String countryName);
+
+        List<String> receiveList();
```

Comment:
maybe would  it be better come up with new more readable name?

Reply:
Ok.